### PR TITLE
Quickly hacked together an example

### PR DESCRIPTION
**Pull request by nikhilpandit**

### CODE REVIEW

**Comment by nikhilpandit** [comment]
@agclever, @ipam73

File: service/db/db.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package db
+
+type Person struct {
```
Comment (ipam73) [inline service/db/db.go:3]: What's the benefit of using newly defined Person struct instead of using the thrift generated Person object?

**Comment by ipam73** [comment]
This looks great @nikhilpandit!  Tried out the example, super easy to follow.

**Comment by agclever** [comment]
I glanced through it very quickly. Looks good over all.

File: service/db/db.go (hunk 1 above)
Comment (nikhilpandit) [inline service/db/db.go:3]: discussed offline (making a note here for history).

**Pull request merged**